pkg/scheduler/factory/factory.go

```diff
@@ -149,10 +149,12 @@ type Configurator interface {
 	GetPredicateMetadataProducer() (predicates.PredicateMetadataProducer, error)
 	GetPredicates(predicateKeys sets.String) (map[string]predicates.FitPredicate, error)
 
-	// Needs to be exposed for things like integration tests where we want to make fake nodes.
-	GetNodeLister() corelisters.NodeLister
 	// Exposed for testing
 	GetClient() clientset.Interface
+
+	// TODO(#80216): Remove these methods from the interface.
```

Comment:
This needs to be addressed according to @bsalamat's comment

Reply:
Not necessarilly, that referred to a different struct.